package.json

```diff
@@ -24,49 +24,54 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-auto-import": "^1.2.21",
-    "ember-cli-babel": "^7.7.3",
-    "git-repo-version": "1.0.2",
-    "lodash.merge": "~4.6.1",
-    "rollbar": "~2.5.5"
+    "ember-auto-import": "^1.5.3",
+    "ember-cli-babel": "^7.13.0",
+    "ember-cli-htmlbars": "^4.2.0",
+    "git-repo-version": "^1.0.2",
+    "lodash.merge": "^4.6.1",
+    "rollbar": "^2.14.4"
   },
   "devDependencies": {
-    "@ember/optional-features": "0.7.0",
-    "bootstrap-sass": "3.4.1",
-    "broccoli-asset-rev": "3.0.0",
-    "ember-bootstrap": "2.6.1",
-    "ember-cli": "~3.9.0",
-    "ember-cli-dependency-checker": "^3.1.0",
+    "@ember/optional-features": "^1.1.0",
+    "@fortawesome/ember-fontawesome": "^0.2.1",
+    "@fortawesome/free-brands-svg-icons": "^5.12.0",
```

Comment:
https://github.com/martndemus/ember-font-awesome hasn't seen much attention since September 2018, so I replaced it with https://github.com/FortAwesome/ember-fontawesome